cli/src/test/java/com/liferay/blade/cli/command/InitCommandTest.java

```diff
@@ -618,7 +618,7 @@ private void _verifyGradleBuild() throws Exception {
 		GradleRunnerUtil.verifyBuildOutput(projectPath.toString(), "foo-1.0.0.jar");
 	}
 
-	private static final String _GRADLE_PLUGINS_WORKSPACE_VERSION = "13.1.0";
+	private static final String _GRADLE_PLUGINS_WORKSPACE_VERSION = "13.1.3";
```

Comment:
@drewbrokke BTW, why the version is `13.1.3`, not `13.1.1`.
How did you get it the correct increased version?